urllib3/util/selectors.py

```diff
@@ -8,9 +8,11 @@
 import errno
 import math
 import select
+import socket
+import sys
 from collections import namedtuple, Mapping
-
 import time
```

Comment:
If we're rearranging imports it might be better to move `import time` above the `from` import.